src/components/PostCard/PostHeader/PostHeader.connector.js

```diff
@@ -73,8 +75,8 @@ export function mergeProps (stateProps, dispatchProps, ownProps) {
     fulfillPost: isCreator ? () => fulfillPost(id) : undefined,
     unfulfillPost: isCreator ? () => unfulfillPost(id) : undefined,
     canFlag: !isCreator,
-    pinPost: canModerate && group ? () => pinPost(id, group.id) : undefined,
-    removePost: !isCreator && canModerate ? () => removePost(id) : undefined,
+    pinPost: (canModerate || responsibilities.includes(RESP_MANAGE_CONTENT)) && group ? () => pinPost(id, group.id) : undefined,
```

Comment:
So, I have left the existing moderation system in place for a few reasons:

1) Large-complex-change redundancy: Maybe something has been missed.
2) Existing group moderators would be locked out of moderator abilities, and unable to add new roles to themselves
3) We still depend on the existing moderator roles with the SurveyStack integration (if that is even operational anymore).

Reply:
hmm, but we just give every current moderator the Manager role and we're good, right? i think we have to do that, leaving the old stuff in will only make life painful moving forward. as far as surveystack goes, i will look into how that works, but again, everything that is currently a moderator just becomes a Manager/Admin

Reply:
While removing the existing moderator setup is a valid goal, I would not try to do it all in one fell swoop.

It would require syncing up mobile and evo releases for a tandem release, something we really don't have the devOps capacity to do. A phased deprecation of the existing moderator setup would be a much smoother experience for users and for us as stewards of the process.

Reply:
I still dont agree with your perspective on this. I don't think we can deploy to production with two overlapping role systems in place. it seems straightforward to me to make all current moderators Managers/Admins and remove the current system. Otherwise we will have to explain and deal with customer support for people navigating the two competing systems and deal with confusing technical debt. Yes syncing up mobile is a pain, but in this case i think we we gotta do it. We do need to implement the ability to change role names though, or just explain how they can make their own role with the same responsibilities and use that ?